deploy.sh

```
@@ -1,7 +1,43 @@
-#!/bin/bash 
+#!/usr/bin/bash
+
+set -o pipefail
+
+shopt -s expand_aliases
+source $HOME/.bash_script_aliases
```

Shellcheck (0.10.0)

[info] Not following: ./.bash_script_aliases was not specified as input (see shellcheck -x).

(SC1091)